src/components/Pagination/Pagination.js

```diff
@@ -300,7 +300,7 @@ export default class Pagination extends Component {
             id={`${prefix}-pagination-select-${inputId}`}
             className={`${prefix}--select__item-count`}
             labelText=""
-            hideLabel
+            noLabel
```

Comment:
This is a breaking change, need to preserve the name or need to introduce a compatibility layer.

Reply:
@asudoh could we look into https://github.com/IBM/carbon-components-react/issues/1662 again for v11?

Reply:
@emyarod Yes, just need an API proposal with research of what components are affected and how.

Reply:
sounds good @asudoh, will update that ticket body with more information from my earlier audits